src/puppetlabs/puppetdb/status.clj

```diff
@@ -0,0 +1,39 @@
+(ns puppetlabs.puppetdb.status
+  (:require [puppetlabs.puppetdb.scf.storage-utils :as sutils]
+            [puppetlabs.puppetdb.utils :as utils]
+            [puppetlabs.puppetdb.mq :as mq]
+            [puppetlabs.puppetdb.schema :as pls]
+            [schema.core :as s]))
+
+(def status-details-schema {:maintenance_mode? s/Bool
```

Comment:
So we're you thinking we'd have a `(def pe-status-details-schema (assoc status-details-schema :rbac-up? <status> :activity-up? <status>)`?